source/administration/changelog.md

```diff
@@ -4,6 +4,118 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
+ - Added a loading animation for "Add Members" channel invite modal.
+ - Made SHIFT+UP switch keyboard focus to right-hand side if it's already open to the current thread.
+ - Removed an unnecessary WebRTC end user setting to avoid user errors and confusion.
+ - Added an on-hover effect for image link previews.
+ 
+ #### Performance
+ - Remove duplicate indexes accidentally created on the ``Channels``, ``Emoji`` and ``OAuthAccessData`` tables.
+ 
+ #### Plugins
+ - Added better plugin error handling and reporting.
+ 
+ #### Slash Commands
+ - Added ``/invite`` slash command to invite users to a channel.
+ - Improved slash command error message when payload has invalid JSON.
+ 
+ #### Administration
+ - Added structured logging to more easily review server logs.
+ - Users' client no longer refreshes after changing a System Console or ``config.json`` setting.
+ 
+ #### Command Line Interface (CLI)
+ - Added `./platform team list` command to list all teams on the server.
+ - Added `./platform permissions reset` command to reset the permissions system to its default state.
+
+#### Enterprise Edition E20
+ - Added cluster event types to [Performance Monitoring](https://docs.mattermost.com/deployment/metrics.html).
+
+### Bug Fixes
+
+ - Fixed an issue where focus with CTRL/CMD+SHIFT+L was always set to the right-hand side when reply thread was open.
+ - Fixed an issue where a user added to a channel wasn't immediately removed from other users' "Add Members" dialog.
+ - Fixed an issue where 'Copy Link' context menu option was partially hidden when right-clicking a team in team sidebar.
+ - Fixed an issue where a user could not log in to Mattermost when their login id ("authdata") failed to migrate properly during migration from LDAP to SAML.
+ - Fixed an issue where plugin configuration was not saved in the System Console.
+
+### Compatibility
+   
+#### Upcoming Deprecated Features in Mattermost v5.0
+
+The following deprecations are planned for the Mattermost v5.0 release, which is scheduled for summer/2018. This list is subject to change prior to the release.
+
+1. All API v3 endpoints will be removed. [See documentation](https://api.mattermost.com/#tag/schema) to learn more about how to migrate your integrations to API v4. [Ticket #8708](https://mattermost.atlassian.net/browse/MM-8708).
+2. `platform` binary will be renamed to mattermost for a clearer install and upgrade experience. All command line tools, including the bulk loading tool and developer tools, will also be renamed from platform to mattermost. [Ticket #9985](https://mattermost.atlassian.net/browse/MM-9985).
+3. [Site URL setting](https://docs.mattermost.com/administration/config-settings.html#site-url) will be enforced to reduce number of OAuth, plugin and email notification set up errors. The setting has already been required since Mattermost v3.8. [Ticket #9983](https://mattermost.atlassian.net/browse/MM-9983).
```

Comment:
A bit unclear. How about "The server will now fail to start if the Site URL setting is not set..."